### DISCUSSION TITLE
Gsoc 2023

### DISCUSSION THREAD

**Discussion by Ankushsup** [koordinates/kart#753] [Google Soc / SoD]
Hi I am ankush, I want to join the group of  Gsoc 2023 , 
Doing contribution to the open source community

I am willing to locate and join the mentor.Excited to work with you.

My tech stack - Java , C++, Git, python, relational database, javascript.

My github  link - https://github.com/Ankushsup?tab=followers

**Comment by rcoup** [comment]
Hi, we haven't decided whether to participate in GSoC 2023 yet... (and if we do, whether we'll be accepted) — check back at the end of February
